chat-plugins/trivia.js

```diff
@@ -727,6 +704,12 @@ var commands = {
 	}
 };
 
+var room = Rooms.get('trivia');
```

Comment:
It should be. The require for ``Rooms`` is run before the one for the command parser. I didn't have any issues with it when I tested the rest of the changes I made.